src/core/dev/hw_queue_tx.h

```diff
@@ -321,7 +325,7 @@ class hw_queue_tx : public xlio_ti_owner {
     // TIS cache. Protected by ring tx lock. TODO Move to ring.
     std::vector<xlio_tis *> m_tls_tis_cache;
 
-    doca_lso_metadata *doca_lso_metadata_head;
+    doca_lso_metadata *m_p_doca_lso_metadata_list = nullptr;
```

Comment:
It would be better to put this var near some other mostly used vars.

Reply:
Done